site/src/docs/components/header/code.mdx

```diff
@@ -15,20 +15,69 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
 
 #### Default
 
-<Playground>
+<Playground scope={{ Header }}>
 
 ```jsx
-<Header>
-  <Header.UniversalBar primaryLinkHref="#" primaryLinkText="Helsingin kaupunki">
-    <Header.NavigationLink href="#">Link</Header.NavigationLink>
-    <Header.NavigationLink href="#">Link</Header.NavigationLink>
-    <Header.NavigationLink href="#">Link</Header.NavigationLink>
-  </Header.UniversalBar>
-</Header>
+import { Header } from 'hds-react';
+
+() => (
+  <>
+    <Header>
+      <Header.UniversalBar primaryLinkHref="#" primaryLinkText="Helsingin kaupunki">
+        <Header.NavigationLink href="#" label="Link" />
+        <Header.NavigationLink href="#" label="Link" />
+        <Header.NavigationLink href="#" label="Link" />
+      </Header.UniversalBar>
+    </Header>
+  </>
+)
```

Comment:
Is the fragment needed? It returns a single child anyway.

Reply:
Not needed. I removed from all Header examples